chrome_extension/content.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file content.js
+ * @description This file is injected into the web page and is responsible for
+ * capturing DOM changes and sending them to the background script.
+ */
+
+let logged = false;
+let ignoreAttributes = new Set();
```

Comment:
Are these used anywhere?

Also this is defined in `contentMutationObserverImplementation.js`, perhaps add a TODO to consolidate?

Reply:
Please remove these if they are not used. If they are used, please add a comment explaining where